examples/regular/adult_wgangp.py

```diff
@@ -1,20 +1,26 @@
-from ydata_synthetic.preprocessing.regular.adult import transformations
+from pmlb import fetch_data
+
 from ydata_synthetic.synthesizers.regular import WGAN_GP
 from ydata_synthetic.synthesizers import ModelParameters, TrainParameters
 
-#Load and process the data
-data, processed_data, preprocessor = transformations()
+model = WGAN_GP
+
+#Load data and define the data processor parameters
+data = fetch_data('adult')
+num_cols = ['age', 'fnlwgt', 'capital-gain', 'capital-loss', 'hours-per-week']
+cat_cols = ['workclass','education', 'education-num', 'marital-status', 'occupation', 'relationship', 'race', 'sex',
+            'native-country', 'target']
 
-# WGAN_GP training
-#Defining the training parameters of WGAN_GP
+# DRAGAN training
+#Defining the training parameters of DRAGAN
 
-noise_dim = 32
+noise_dim = 128
 dim = 128
-batch_size = 128
+batch_size = 500
 
 log_step = 100
 epochs = 300+1
-learning_rate = [5e-4, 3e-3]
+learning_rate = 1e-5
```

Comment:
Why have we changed this parameter to a single instead of a tuple as it was in the example? 

If I recall correctly this was meant to showcase that we can provide different learning rates for the Generator and Discriminator.